eng/pipelines/coreclr/templates/build-corelib-job.yml

```diff
@@ -0,0 +1,163 @@
+parameters:
+  archType: ''
+  buildConfig: ''
+  compilerName: 'clang'
+  condition: true
+  container: ''
+  crossrootfsDir: ''
+  isOfficialBuild: false
+  osGroup: ''
+  osSubgroup: ''
+  platform: ''
+  pool: ''
+  signBinaries: false
+  stagedBuild: false
+  testGroup: ''
+  timeoutInMinutes: ''
+  variables: {}
+
+### Product build
+jobs:
+- template: xplat-pipeline-job.yml
+  parameters:
+    buildConfig: ${{ parameters.buildConfig }}
+    _BuildConfig: ${{ parameters.buildConfig }}
+    archType: ${{ parameters.archType }}
+    osGroup: ${{ parameters.osGroup }}
+    osSubgroup: ${{ parameters.osSubgroup }}
+    testGroup: ${{ parameters.testGroup }}
+    helixType: 'build/product/'
+    enableMicrobuild: true
+    stagedBuild: ${{ parameters.stagedBuild }}
+    pool: ${{ parameters.pool }}
+    condition: ${{ parameters.condition }}
+
+    # Compute job name from template parameters
```

Comment:
wonder if these options are required for corelib

Reply:
We still need the names for osGroup, arch, and config for sure. We may still need osSubgroup. We don't need compiler though.

Also, can we change the job display names to say that these are CoreLib jobs? Makes it a lot easier to read on the AzDO view.

Reply:
cleaned up some parameters and also updated display names.